tests/Behat/Context/Api/DashboardContext.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\CommerceWeavers\SyliusAlsoBoughtPlugin\Behat\Context\Api;
+
+use Behat\Behat\Context\Context;
+
+final class DashboardContext implements Context
+{
+    /**
+     * @When I open the :channel channel dashboard
+     */
+    public function iOpenTheChannelDashboard(string $channelName): void
+    {
+        // intentionally left blank
+    }
+}
```

Comment:
This doesn't seem to be used anywhere; the scenarios using this step don't have the `@api` tag